client/state/data-layer/wpcom/activity-log/rewind/restore-status/index.js

```diff
@@ -50,7 +50,7 @@ export const receiveRestoreProgress = ( { dispatch }, { siteId, timestamp, resto
 };
 
 // FIXME: Could be a network Error (instanceof Error) or an API error. Handle each case correctly.
-export const receiveRestoreError = ( { dispatch }, { siteId, timestamp, restoreId }, error ) => {
+export const receiveRestoreError = ( { dispatch }, {}, error ) => {
```

Comment:
I didn't know it's possible to specify an empty destructuring ;) It's better to name the parameter:
```js
( { dispatch }, action, error ) => ...
```

Reply:
Automattic motto: always learning! :D Thanks for catching this, it's fixed now.